doc/rst/tools/mason/start/installation.rst

```diff
@@ -3,10 +3,34 @@
 Installation Instructions
 =========================
 
+If you have installed Chapel from a package manager or binary distribution,
+mason may already be included. You can check by running ``mason --version`` in
+your terminal.
+
+Building and Installing Mason From Source
+-----------------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+To build mason from source, ensure you have a working Chapel build from source
+and that your environment is set up correctly, as described in
+:ref:`readme-chplenv`.
+
+Mason requires a build of Chapel with ``CHPL_COMM=none`` and
```

Comment:
Would you be up for spinning an issue for using/building Mason with CHPL_COMM?

Reply:
Done: https://github.com/chapel-lang/chapel/issues/28046